pytest_mh/utils/fs.py

```diff
@@ -230,6 +231,23 @@ def exists(self, path: str) -> bool:
 
         return False
 
+    def selinux_context(self, path: str) -> SELinuxContext | None:
+        """
+        Checks SELinux file context.
+
+        :param path: File path.
+        :type path: str
+        :return: SELinux file context
+        :rtype: SELinuxContext or None
+        """
+        self.logger.info(f'Checking SELinux context for "{path}"')
```

Comment:
The same - 'Reading'.

Reply:
I prefer 'Gets' and 'Getting', so I updated it accordingly